ndcube/asdf/resources/schemas/resampledwcs-0.1.0.yaml

```diff
@@ -0,0 +1,23 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/resampledwcs-0.1.0"
+
+title:
+  Represents the ndcube ResampledLowLevelWCS object
+
+description:
+  Represents the ndcube ResampledLowLevelWCS object
+
+type: object
+properties:
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  factor:
+    type: object
+  offset:
+    type: object
```

Comment:
Yes, these values can be numbers, but since there isn't any public method to retrieve the values of `factor` and `offset`, we have to access the private attributes `_factor` and `_offset`. These attributes are instances of the `np.array` class(`factor` and `offset` are converted [here](https://github.com/sunpy/ndcube/blob/8a4c09a3f6f84768734587621531b83bf6d1be06/ndcube/wcs/wrappers/resampled_wcs.py#L32)), so the schema checks if they are of type "object." 

However, it might be better to check against the specific tag: "tag:stsci.edu:asdf/core/ndarray-1.0.0". Thanks for pointing it out.